pkgs/servers/mautrix-telegram/default.nix

```diff
@@ -33,6 +33,11 @@ in python.pkgs.buildPythonPackage rec {
 
   patches = [ ./0001-Re-add-entrypoint.patch ];
 
+  postPatch = ''
+    substituteInPlace requirements.txt \
+      --replace "asyncpg>=0.20,<0.27" "asyncpg>=0.20"
+  '';
```

Comment:
Can't we use `pythonRelaxDepsHook`, it is more robust is less error-prone.

Reply:
Nope, it doesn't work: https://gist.github.com/azahi/27bda72429393ee5a139b776be8ba777
I did
```nix
nativeBuildInputs = [ python.pkgs.pythonRelaxDepsHook ];
pythonRelaxDeps = [ "asyncpg" ];
```

Reply:
Looks like mautrix-telegram is doing funky things with version numbers.